python/core/auto_generated/geometry/qgsgeometry.sip.in

```diff
@@ -1943,7 +1943,9 @@ by calling :py:func:`~QgsGeometry.lastError` on the returned geometry.
 
 .. note::
 
-   Ported from PostGIS ST_MakeValid() and it should return equivalent results.
+   For QGIS builds using GEOS library versions older than 3.9 this method calls
```

Comment:
GEOSMakeValid_r was actually introduced in GEOS 3.8

Reply:
Fixed